File: Beecrowd/1873 Pedra-papel-tesoura-lagarto-Spock/main.cpp
#include <iostream>

using namespace std;

int main(){
    int c;
    string jog1, jog2;

    cin >> c;

    while(c--){
        cin >> jog1 >> jog2;

        if((jog1 == "tesoura" and jog2 == "papel") or (jog1 == "tesoura" and jog2 == "lagarto") or 
            (jog1 == "papel" and jog2 == "pedra") or (jog1 == "papel" and jog2 == "spock") or
            (jog1 == "pedra" and jog2 == "lagarto") or (jog1 == "pedra" and jog2 == "tesoura") or
            (jog1 == "lagarto" and jog2 == "spock") or (jog1 == "lagarto" and jog2 == "papel") or
            (jog1 == "spock" and jog2 == "tesoura") or (jog1 == "spock" and jog2 == "pedra"))
                cout << "rajesh" << endl;
        else if((jog2 == "tesoura" and jog1 == "papel") or (jog2 == "tesoura" and jog1 == "lagarto") or 
            (jog2 == "papel" and jog1 == "pedra") or (jog2 == "papel" and jog1 == "spock") or
            (jog2 == "pedra" and jog1 == "lagarto") or (jog2 == "pedra" and jog1 == "tesoura") or
            (jog2 == "lagarto" and jog1 == "spock") or (jog2 == "lagarto" and jog1 == "papel") or
            (jog2 == "spock" and jog1 == "tesoura") or (jog2 == "spock" and jog1 == "pedra"))
                cout << "sheldon" << endl;
        else
            cout << "empate" << endl;
    }

    return 0;
}

File: Beecrowd/1873 Pedra-papel-tesoura-lagarto-Spock/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int c;
        String jog1, jog2;

        c = sc.nextInt();

        while (c > 0) {
            jog1 = sc.next();
            jog2 = sc.next();

            if ((jog1.equals("tesoura") && jog2.equals("papel")) || (jog1.equals("tesoura") && jog2.equals("lagarto"))
                    ||
                    (jog1.equals("papel") && jog2.equals("pedra")) || (jog1.equals("papel") && jog2.equals("spock")) ||
                    (jog1.equals("pedra") && jog2.equals("lagarto")) || (jog1.equals("pedra") && jog2.equals("tesoura"))
                    ||
                    (jog1.equals("lagarto") && jog2.equals("spock")) || (jog1.equals("lagarto") && jog2.equals("papel"))
                    ||
                    (jog1.equals("spock") && jog2.equals("tesoura")) || (jog1.equals("spock") && jog2.equals("pedra")))
                System.out.println("rajesh");
            else if ((jog2.equals("tesoura") && jog1.equals("papel"))
                    || (jog2.equals("tesoura") && jog1.equals("lagarto")) ||
                    (jog2.equals("papel") && jog1.equals("pedra")) || (jog2.equals("papel") && jog1.equals("spock")) ||
                    (jog2.equals("pedra") && jog1.equals("lagarto")) || (jog2.equals("pedra") && jog1.equals("tesoura"))
                    ||
                    (jog2.equals("lagarto") && jog1.equals("spock")) || (jog2.equals("lagarto") && jog1.equals("papel"))
                    ||
                    (jog2.equals("spock") && jog1.equals("tesoura")) || (jog2.equals("spock") && jog1.equals("pedra")))
                System.out.println("sheldon");
            else
                System.out.println("empate");

            c--;
        }

        sc.close();
    }

}